### ISSUE TITLE
How to force RF to use a specific (different) sub folder every time the page is loaded?

### ISSUE THREAD

**Issue by newness2017** [trippo/ResponsiveFilemanager#359]
I want RF to check the session variable ($_SESSION['RF']['subfolder']) every time the page is loaded.
Each page in my website requires uploading to different folders and I do not want users to get confused and upload to the wrong place.
Currently, RF seems to read the session variable once and then refer to itself for every page load after that. So if a user is at page X (session variable says /uploads/page-x) and then navigates to page Y (session variable says /uploads/page-y), RF still returns to /uploads/page-x when loaded and I want it to return to /uploads/page-y
How can I achieve this?
Thanks

**Comment by newness2017** [comment]
Not sure if this is an ideal 'fix' but I've edited the dialog.php file at line 90 to read:
`<?php $rfm_subfolder = $_COOKIE['RF_subfolder_var']; ?>`
Then in my index.php file I set the cookie appropriate to the page URL and when I execute RF from TinyMCE, it opens at the correct subfolder.
One thing I have noticed by using this method is the subfolder is 'seen' as the root folder, so I cannot navigate 'up' to the 'uploads' folder - which is my preference anyway! But when I was experimenting with the session variable, although it jumped into the subfolder, I could navigate up to the root folder 'uploads'.
Food for thought.
